<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dominioEspacial</class>
 <widget class="QDialog" name="dominioEspacial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dominio Espacial</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Imagen</string>
   </property>
   <widget class="QPushButton" name="pbtnCargar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cargar</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblImagen">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>30</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>111</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Espacio de color</string>
   </property>
   <widget class="QRadioButton" name="rbtnRGB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>solo RGB</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtnGray">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Escala de grises</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtnYCrCb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>YCrCb</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtnYUV">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>YUV</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Proceso</string>
   </property>
   <widget class="QRadioButton" name="rbtnIncrustar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Incrustar</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtnExtraer">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Extraer</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>181</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Descripcion</string>
   </property>
   <widget class="QPlainTextEdit" name="ptxtDescripcion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>571</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Mensaje</string>
   </property>
   <widget class="QPlainTextEdit" name="ptxtMensaje">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>541</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>121</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Acci√≥n</string>
   </property>
   <widget class="QPushButton" name="pbtnProcesar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Procesar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtnSalir">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
  </widget>
  <widget class="QProgressBar" name="pBarra">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>301</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbtnSalir</sender>
   <signal>clicked()</signal>
   <receiver>dominioEspacial</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>528</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>591</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
